ansible_collections/arista/avd/molecule/eos_validate_state/reports/FABRIC-state.md

```diff
@@ -44,15 +44,14 @@
 | Test Category | Total Tests | Tests Passed | Tests Failed | Tests Skipped |
 | ------------- | ----------- | ------------ | ------------ | ------------- |
 | AAA | 133 | 0 | 0 | 0 |
+| BFD | 6 | 0 | 0 | 0 |
```

Comment:
Looks like we are keeping our own list of acronyms to upper case? I think there is already some place in AVD we can leverage for this.

Reply:
There is the `plugins\plugin_utils\schema\key_to_display_name.py` module but I'm not sure if we should leverage this since this is for the schema which is irrelevant for validate state.

Reply:
ack lets close it for now but sad to be maintaining it twice :p